packages/signers/meta-signers/src/multisig-signer.ts

```diff
@@ -55,19 +53,23 @@ export function getMultisigSigner(
     }
   }>,
   signer: PolkadotSigner | WrappedSigner,
-  options?: MultisigSignerOptions,
-): PolkadotSigner {
+  options?: MultisigSignerOptions<Address>,
+): WrappedSigner {
   options = {
     ...defaultMultisigSignerOptions,
     ...options,
   }
 
+  const toSS58 = AccountId().dec
+  const toAddress = (value: Uint8Array): Address => {
+    if (multisig.signatories[0].startsWith("0x")) {
+      return toHex(value) as Address
+    }
+    return toSS58(value) as Address
+  }
```

Comment:
Dunno if it could be possible in the future Plaza that both `AccountId20` and `AccountId32` will be supported. In that case, multisigs could have mixed addresses as signatories, right?

Reply:
Not sure, I don't think that's the case - This `toAddress` is just used in two places:

- `codec.enc({ other_signatories: … })`, where the codec transformes it back to the Uint8Arr. In order for this to be problematic, it would mean that the Multisig.as_multi would have to take a MultiAddress instead of an AccountId.
- `getMultisigInfo(multisig: …)` where we just pass the multisig address in the AccountId format of the chain - this is what's passed to the storage, and same reasoning as above.

So this isn't a concern while the multisig pallet doesn't work with MultiAddress. If they do change this, then we will need to refactor it.